packages/components/src/inputs/tests/chromatic/PasswordField.stories.tsx

```diff
@@ -112,7 +110,7 @@ export const Zoom: Story = {
 export const Styling: Story = {
     render: () => (
         <Inline>
-            <PasswordField border="core_amanita-600" aria-label="Label" />
+            <PasswordField UNSAFE_border="1px solid red" aria-label="Label" />
```

Comment:
Why do you use `UNSAFE_border`?

Reply:
due to a previous comment left by Francis.
since these are chromatic tests, he said that we should use a static color so that if the colors in hopper are ever updated, the test won't break. The only way to use this is to use the UNSAFE one. unless I use style. I'm pretty sure I've seen this used before though.

Reply:
it makes sense I understand better